src/bootstrap/util.rs

```diff
@@ -45,7 +45,6 @@ pub fn copy(src: &Path, dst: &Path) {
                dst.display(), e)
     }
     let metadata = t!(src.metadata());
-    t!(fs::set_permissions(dst, metadata.permissions()));
```

Comment:
This change seems unrelated?

Reply:
Ah, right. It's not needed, but I forgot to remove it from this patch.